amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/preview/PreviewActivityService.java

```diff
@@ -374,7 +375,9 @@ private Double convertProjectCostAmount(AmpFundingAmount projectCost, String cur
     
     public boolean isViewWorkspacesButtonVisible(Long ampActivityId) throws DgException {
         AmpActivityVersion activity = ActivityUtil.loadActivity(ampActivityId);
-        return ActivityUtil.isTeamMemberValidator(TeamMemberUtil.getLoggedInTeamMember(), activity);
+        TeamMember teamMember = TeamMemberUtil.getLoggedInTeamMember();
+        
+        return teamMember.isApprover() || ActivityUtil.isTeamMemberValidator(teamMember, activity);
```

Comment:
So, the fact that a tm is approver doesn't mean it can validate an activity through cross team validation.
"Cross Team Validation checkbox enables the workspace manager and/or approver to Validate activities that were created on other workspaces linked to the workspace where they are logged in."

That's why the condition was taken out from isTeamMemberValidator method.